.github/workflows/test-acceptance.yaml

```diff
@@ -61,6 +61,10 @@ jobs:
         CYPRESS_DEFAULT_COMMAND_TIMEOUT: 40000
         CYPRESS_PAGE_LOAD_TIMEOUT: 120000
         CYPRESS_RETRIES: 3
+        CYPRESS_PASSWORD: password
+        PASSWORD: password
+        USE_AUTH: true
+        NODE_ENV: production
```

Comment:
Note to self: `NODE_ENV` is needed because otherwise `USE_AUTH: true` has no effect.